jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -385,6 +385,14 @@ def check_compute_capability(capability):
   current = tuple(int(x) for x in d.compute_capability.split("."))
   return current >= target
 
+def is_cuda_compute_capability_equal(capability):
+  if not 'cuda' in xla_bridge.get_backend().platform_version:
+    return False
+  d, *_ = jax.local_devices(backend="gpu")
+  target = tuple(int(x) for x in capability.split("."))
+  current = tuple(int(x) for x in d.compute_capability.split("."))
```

Comment:
Not 100% sure, but can't it be 9.0a?

Reply:
I think XLA only returns 9.0 for hopper compute capability, 9.0a is only used for PTX compatibility.